common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -108,6 +108,24 @@ object Mappings {
       "filename" -> nonAnalyzedString
     )
 
+  val usageReference =
+    nonDynamicObj(
+      "referenceType" -> nonAnalyzedString,
+      "uri"  -> nonAnalyzedString,
+      "name" -> sStemmerAnalysedString
+    )
+
+  val usagesMapping =
+    nonDynamicObj(
+      "title"        -> sStemmerAnalysedString,
```

Comment:
I would imagine you'd want some analysis on it - not `non_analysed` so you could match on a partial title?